package.json

```diff
@@ -546,6 +546,87 @@
         }
       }
     ],
+    "walkthroughs": [
+      {
+        "id": "TektonWalkthrough",
+        "title": "Getting Started with Tekton Pipeline",
+        "description": "Start creating, running, and managing your Tekton Pipelines on OpenShift and Kubernetes",
+        "steps": [
+          {
+            "id": "Create Resources",
+            "title": "Create new resource",
+            "description": "Create a new Tekton resource directly from the YAML in the editor and deploy it using the `Kubernetes: Create` command.",
+            "media": {
+              "image": "https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/walkthroughs/create-resource.gif",
```

Comment:
Why are we using the raw location of the images? This should point to the local folder image.